microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/jaxrs/java/JaxRsCodeLensParticipant.java

```diff
@@ -72,18 +72,20 @@ public boolean isAdaptedForCodeLens(JavaCodeLensContext context, IProgressMonito
 	public List<CodeLens> collectCodeLens(JavaCodeLensContext context, IProgressMonitor monitor) throws CoreException {
 		ITypeRoot typeRoot = context.getTypeRoot();
 		IJavaElement[] elements = typeRoot.getChildren();
-		int serverPort = JaxRsContext.getJaxRsContext(context).getServerPort();
+		JaxRsContext jaxrsContext = JaxRsContext.getJaxRsContext(context);
+		int serverPort = jaxrsContext.getServerPort();
 		IJDTUtils utils = context.getUtils();
 		MicroProfileJavaCodeLensParams params = context.getParams();
 		params.setLocalServerPort(serverPort);

```

Comment:
Remove setLocalServerPort from MicroProfileJavaCodeLensParams

Reply:
I think we should keep the setter. Also MicroProfileDelegateCommandHandlerForJava.java line 228 uses it.

Reply:
Ok keep like this, we will remove it when we will implement https://github.com/eclipse/lsp4mp/issues/163, but please remove code:

```java
int serverPort = jaxrsContext.getServerPort();
MicroProfileJavaCodeLensParams params = context.getParams();
params.setLocalServerPort(serverPort);
```